html/themes/custom/common_design_subtheme/templates/node--operation-tab.html.twig

```diff
@@ -31,6 +31,13 @@
     </div>
   {% endif %}
 
+  {% if can_edit %}
+    {{ attach_library('common_design_subtheme/node-edit-link') }}
+    <div class="node-edit--link">
+      <a href="{{ path('entity.node.edit_form', {'node': node.id}) }}?destination={{ path('<current>') }}">{{ 'Edit this page'|t }}</a>
```

Comment:
```suggestion
      <a href="{{ path('entity.node.edit_form', {'node': node.id}) }}?destination={{ path('<current>') }}"><span class="visually-hidden">{{ 'Edit this page'|t }}</span></a>
```

Marking text as visually-hidden still allows icon to appear on hover without overlap text:

<img width="1180" alt="Screen Shot 2022-09-05 at 14 21 56" src="https://user-images.githubusercontent.com/254753/188448142-5519b3a4-24af-4f61-afbe-143228d1378b.png">

Reply:
The icon isn't from this patch

Reply:
gotcha, ignore me